src/resources/views/crud/columns/multidimensional_array.blade.php

```diff
@@ -1,16 +1,20 @@
 {{-- enumerate the values in an array  --}}
```

Comment:
In my tests, when `multidimensional_array` is empty this will output nothing, not `-` as I expected. This happened before this PR, and this PR doesn't make it show `-` like other columns do.

Reply:
This one is also fixed 🙌